docs/aws-professional/services.md

```diff
@@ -22,168 +22,168 @@ For an overview of Azure for AWS users, see [Introduction to Azure for AWS exper
 
 
-## Management & monitoring
+## Management & Monitoring
 
-| Area                              | AWS service                       | Azure service                                                                                                                                                                                                                                                                             | Description                                                                                                                                                                                                                               |
+| Service                              | AWS                        | Azure                                                                                                                                                                                                                                                                              | Description                                                                                                                                                                                                                               |
 |-----------------------------------|-----------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| Cloud advisor                     | Trusted Advisor                   | [Azure Advisor](https://azure.microsoft.com/services/advisor/)                                                                                                                                                                                                                                                                                      | Provides analysis of cloud resource configuration and security so subscribers can ensure they’re making use of best practices and optimum configurations.                                                                                 |
-| Deployment orchestration (DevOps) | OpsWorks (Chef-based)             | [Azure Automation](https://azure.microsoft.com/services/automation/)                                                                                                                                                                                                                | Con
```

Comment:
Two while clauses in one sentence is awkward, can we re-word this or revert to two sentences?

Reply:
Sure. I don't see the while you mentioned or I'd help provide a suggestion.